scripts/tests/run_cxx_ps_for_py_ps_tests.sh

```diff
@@ -3,4 +3,4 @@ set -eo pipefail
 
 source "$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)/../set_vars.sh"
 
-"$METASYSTEM_PATH"/scripts/run_sc_server.sh --repo_path "${APP_ROOT_PATH}"/kb.bin-tests --clear -p 8091 &
+"${METASYSTEM_PATH}/scripts/run_sc_machine.sh" --repo_path "${APP_ROOT_PATH}/kb.bin-tests" --clear -p 8091 &
```

Comment:
Wait, will this even work? py-kpm depends on py-sc-client which, in turn, depends on sc-server